main.js

```diff
@@ -1 +1,13 @@
-
+function dis (value){
+    document.getElementById("edu").value+=value;
+    }
+    
+    function solve(){
+    let x = document.getElementById("edu").value;
+    let y = eval(x);
+    document.getElementById("edu").value = y;
+    }
+    
+    function clr(){
+    document.getElementById("edu").value = ""
+    }
```

Comment:
Great work on this Michael, looking forward to seeing how you continue to work with DOM manipulation as we move along. Keep pushing